KsApi/mutations/inputs/ApplePayParams.swift

```diff
@@ -1,17 +1,20 @@
 import Foundation
 
 public struct ApplePayParams: Encodable, Equatable {
+  public let paymentMethodId: String?
```

Comment:
Xcode was yelling at me that this property was 'inaccessible' due to the struct being 'internal', even though it and its init are both marked as public. Explicitly labeling this method as public was the only way to get around it for me.